x/incentives/keeper/gauge_test.go

```diff
@@ -507,7 +507,8 @@ func (s *KeeperTestSuite) TestCreateGauge_NoLockGauges() {
 			distrTo: lockuptypes.QueryCondition{
 				LockQueryType: lockuptypes.NoLock,
 				// Note: this assumes the gauge is external
-				Denom: "",
+				Denom:    "",
```

Comment:
Can we also add non-failing cases for the three branches that has been added to nolock createGauge?

Reply:
The non-failing case should be covered by existing tests, but will re review to make sure I'm not missing anything

Reply:
Sg! On first site of existing tests, test cases did not seem obvious in which case touches which branch. Please feel free to merge once you have re-checked!

Reply:
Right – since 1 nanosecond is an authorized uptime by default, the test cases with that duration should be covering the non-failing branches